database/mongo.index.js

```diff
@@ -0,0 +1,57 @@
+//USING NATIVE MONGODB DRIVER
+var MongoClient = require('mongodb').MongoClient;
+var url = "mongodb://localhost:27017/bookshelf";
+var bookshelf;
+
+MongoClient.connect(url,function(err,client){
+    console.log('connected');
+    bookshelf= client.db('bookshelf')
+
+  }
+)
+
+
+var getBooks =  async (booksId)=>{
+
+  var booksId = Number(booksId)
+  var books = await bookshelf.collection('books').findOne({booksId:booksId})
```

Comment:
Curious to know if this await is needed. Im also not 100% sure how await fully works.